api/src/pcapi/routes/native/v2/serialization/bookings.py

```diff
@@ -38,12 +38,21 @@ class BookingVenueResponseV2GetterDict(GetterDict):
     def get(self, key: str, default: Any | None = None) -> Any:
         if key == "name":
             return self._obj.common_name
+        if key == "address":
+            return BookingVenueResponseAddressV2(
+                id=self._obj.offererAddress.address.id if self._obj.offererAddress else None
+            )
 
         return super().get(key, default)
 
 
+class BookingVenueResponseAddressV2(ConfiguredBaseModel):
```

Comment:
```suggestion
class BookingVenueAddressResponseV2(ConfiguredBaseModel):
```
Peitt détail de naming